### ISSUE TITLE
Add 1 to 1 chat functionality

### ISSUE THREAD

**Issue by tomazbracic** [dwyl/phoenix-chat-example#51]
It would be super awesome to upgrade this great tutorial with a bit more advanced example. How would one pick up one person from (presence?) and open "private" 1-to-1 chat with it. On prior approval.

**Comment by nelsonic** [comment]
This would be fun to add but relies on `presence` and `auth` working. 💭 
@LuchoTurtle want to take a look at this after #149 ?

**Comment by nelsonic** [comment]
This functionality is a (_very_) nice-to-have. 
We can revisit it once our `App` is shipped! 👌
